src/index.js

```diff
@@ -12,48 +12,41 @@ export default function loader(content) {
 
   const context = options.context || this.rootContext || (this.options && this.options.context);
 
-  let url = loaderUtils.interpolateName(this, options.name, {
+  const originalURL = loaderUtils.interpolateName(this, options.name, {
     context,
     content,
     regExp: options.regExp,
   });
 
-  let outputPath = '';
-
-  if (options.outputPath) {
-    // support functions as outputPath to generate them dynamically
-    outputPath = (
-      typeof options.outputPath === 'function' ? options.outputPath(url) : options.outputPath
-    );
-  }
+  let outputPath = (
+    typeof options.outputPath === 'function' ? options.outputPath(originalURL) : path.join((options.outputPath || '') + originalURL)
```

Comment:
```js
: path.join(options.outputPath || '', url)
```

Please use a multiline ternary for readability here

```js
 typeof x === 'function'
  ? y
  : z
```

Reply:
@michael-ciniawsky eslint fix, old `webpack-default`

Reply:
`eslint-disable`